docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
I understand what you are saying. We will continue to have /etc/sysconfig/docker-storage-setup which already has couple of lines and we can add more.

Key question is, by retaining /etc/sysconfig/docker-storage, we are promoting users to pass in dm.metadatadev and dm.datadev and that's a mode we dss does not support and we don't want to support.

So even if we want to retain the file, that's fine but atleast we should cleanup the text and get rid of mentions of doing dm.datatdev and dm.metadatadev.